client-v2/src/keyboard/index.ts

```diff
@@ -13,11 +13,18 @@ import paste from './keyboardActionMaps/paste';
 import { moveUp, moveDown } from './keyboardActionMaps/move';
 import { ArticleFragment } from '../shared/types/Collection';
 
-type FocusableTypes = 'clipboard' | 'clipboardArticle';
+type FocusableTypes =
```

Comment:
We could express BaseFocusState as an intersection of different `FocusableTypes` and their unique properties here, a bit like Action types in redux -- this might save us some null checking later on! Not a blocker but might be handy 👍